iothub/service/samples/PnpServiceSamples/SimpleThermostat/Program.cs

```diff
@@ -3,7 +3,7 @@
 using Microsoft.Azure.Devices;
 using Microsoft.Azure.Devices.Shared;
 
-namespace PnpServiceSample
+namespace Thermostat
 {
     public class Program
```

Comment:
the command impl needs to be updated here, `Thermostat` does not have "reboot" command.

Reply:
Yup, aware of that. I have not changed it in this PR just changing namespace